src/lib/y2partitioner/widgets/fstab_options.rb

```diff
@@ -0,0 +1,538 @@
+require "yast"
+require "cwm"
+require "y2storage"
+
+module Y2Partitioner
+  module Widgets
+    # The fstab options are mostly checkboxes and combo boxes that share some
+    # commong methods, so this is a mixin for that share code.
+    module FstabCommon
+      def initialize(options)
+        textdomain "storage"
+
+        @options = options
+      end
+
+      def init
+        init_regexp if self.class.const_defined?("REGEXP")
+      end
+
+      # No all the fstab options are supported by all the filesystem so each
+      # widget are able to check if the current filesystem is supported
+      # explicitely or checking if the values it is responsable of are
+      # supported by the filesystem.
+      def supported_by_filesystem?
+        return false if !@options.filesystem_type
+
+        if self.class.const_defined?("SUPPORTED_FILESYSTEMS")
+          self.class::SUPPORTED_FILESYSTEMS
+            .include?(@options.filesystem_type.to_sym)
+        else
+          self.class::VALUES.all? do |v|
+            @options.filesystem_type.supported_fstab_options.include?(v)
+          end
+        end
+      end
+
+      # @param widget [CWM::AbstractWidget]
+      # @return [Boolean] true if it is supported by current filesystem
+      def ui_term?(widget)
```

Comment:
Argh! This method simply obscures the perfectly good name of the method which it calls. Drop it.